edx_proctoring/views.py

```diff
@@ -168,6 +170,68 @@ def handle_exception(self, exc):
         return resp
 
 
+class ProctoredExamAttemptsMFEView(ProctoredAPIView):
```

Comment:
I don't like the idea of implying this view is MFE specific since it encourages coupling to a specific use case. Something like `ProctoredExamAttemptView` it would be better for future use cases.

Reply:
fixed